apps/projects/app/components/Shared/FilterBar/FilterBar.js

```diff
@@ -75,6 +75,46 @@ SearchPopover.propTypes = {
   updateTextFilter: PropTypes.func.isRequired,
 }
 
+const SortPopover = ({ visible, opener, setSortMenuVisible, sortBy, updateSortBy }) => {
+  const theme = useTheme()
+  const sorters = [
+    'Name ascending',
+    'Name descending',
+    'Newest',
+    'Oldest',
+  ]
```

Comment:
No need for this to be defined within the component. This causes it to be garbage-collected and re-allocated over and over.

Reply:
Moved to the top.